src/components/map/Graph.js

```diff
@@ -51,106 +54,103 @@ const Title = styled.div`
   padding: 5px;
 `;
 
-@inject(app("Filters", "UI"))
-@observer
-class Graph extends React.Component {
```

Comment:
Refactored to function component with hooks so we can use useMemo and useEffect. Also improves state handling with useState.